src/primitives/transaction.h

```diff
@@ -992,6 +992,10 @@ class CTransaction
         return a.wtxid.hash != b.wtxid.hash;
     }
 
+    int64_t EstimatePropagationLatencyInSeconds() const {
+        return 10;
```

Comment:
I think it'd be better for this to be directly controlled by the startup parameter.